src/plugin.js

```diff
@@ -121,6 +121,14 @@ const monitorPlayback = function(player, options) {
         resetMonitor();
       }
     });
+    healthcheck('adtimeupdate', function() {
```

Comment:
the healthcheck function could probably be reused in both places.
Also, `player`'s `on` can take an array of things to listen on, so, `healthcheck` itself could probably just be `healthcheck(['timeupdate', 'adtimeupdate'], ...)`

Also, is there any way to fix this without making errors aware of ads?